.github/workflows/code-references.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Collect code references
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  refs:
+    name: Code reference collection
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Collect
+        id: collect
+        uses: launchdarkly/find-code-references-in-pull-request@2e9333c88539377cfbe818c265ba8b9ebced3c91 # v1.1.0
+        with:
+          project-key: default
+          environment-key: dev
+          access-token: ${{ secrets.LD_ACCESS_TOKEN }}
```

Comment:
Is this secret in the repository?  Should it be in a Key Vault instead?

Reply:
Global secret in GitHub, for all repositories. We have a few.